stdlog.go

```diff
@@ -74,6 +80,14 @@ func (s *stdlogAdapter) pickLevel(str string) (Level, string) {
 	}
 }
 
+func (s *stdlogAdapter) trimTimestamp(str string) string {
+	// Ignore characters commonly found in timestamp formats from the beginning
+	// of the input.
+	logTimestampRegexp := regexp.MustCompile(`^[\d\s\:\/\.\+-TZ]*`)
```

Comment:
Can you move this out to be a global so it's not being compiled on every call?

Reply:
Ah good call. Will do!